### PR TITLE
add P136 anticompact to S55

### PR DESCRIPTION
**Pull request by StevenClontz**

### CODE REVIEW

**Comment by StevenClontz** [comment]
See discussion at https://math.stackexchange.com/questions/4580731/is-there-a-connected-t-1-anticompact-topological-space

**Comment by prabau** [comment]
Nice.  I did not realize that space was in "Counterexamples".

File: spaces/S000055/properties/P000136.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+property: P000136
+value: true
+refs:
+  - mathse: 4580731
+    name: Is there a connected T1 anticompact topological space?
+  - doi: 10.1007/978-1-4612-6290-9 
+    name: Counterexamples in Topology
+---
+
+Item 7 of space 63 in {{doi:10.1007/978-1-4612-6290-9}}, see also {{mathse:4580731}}.
```
Comment (prabau) [inline spaces/S000055/properties/P000136.md:12]: Really nitpicking here, but for consistency with other spaces should we write "item #7 in space #63" ?

**Review by prabau — APPROVED** [review]

File: spaces/S000055/properties/P000136.md (hunk 1 above)
Comment (StevenClontz) [reply spaces/S000055/properties/P000136.md:12]: Sure, I'll change that when I'm back in the office next week.

**Comment by StevenClontz** [comment]
> Nice. I did not realize that space was in "Counterexamples".

To be honest I almost made a PR to add it before I found it. 😅

**Pull request merged**